src/components/Visualizations/inflictorWithValue.jsx

```diff
@@ -239,14 +239,14 @@ class InflictorWithValue extends React.Component {
               {value > 0 && value}
             </div>
           }
+            {charges &&
             <div className='textOverlay'>
-              {charges &&
               <div className="chargeOverlay">
                 {charges}
               </div>
-              }
               <div className="overlay">{value}</div>
```

Comment:
aren't we rendering the value already in line 239? Not sure if this is needed here

Reply:
I think you're right.